src/main/scala/org/clulab/reach/ReachShell.scala

```diff
@@ -1,16 +1,19 @@
 package org.clulab.reach
 
 import java.io.File
+
 import com.typesafe.config.ConfigFactory
 import jline.console.ConsoleReader
 import jline.console.history.FileHistory
+
 import scala.collection.immutable.ListMap
 import org.clulab.reach.display._
 import org.clulab.reach.context.ContextEngineFactory.Engine
 import RuleReader._
+import com.typesafe.scalalogging.LazyLogging
 
-object ReachShell extends App {
-  println("Loading ReachSystem ...")
+object ReachShell extends App with LazyLogging {
```

Comment:
Sorry about not being more specific when we discussed this issue, but the `ReachShell` should be using `println`, not `logger.info`.  Can you please revert your changes to this file?